hibernate-core/src/main/java/org/hibernate/proxy/pojo/bytebuddy/SerializableProxy.java

```diff
@@ -30,6 +30,8 @@ public final class SerializableProxy extends AbstractSerializableProxy {
 
 	private final CompositeType componentIdType;
 
+	private static BytecodeProviderImpl bytecodeProvider;
```

Comment:
should this be `volatile`?

Reply:
Sure, I guess one could be doing this concurrently.